src/EditorFeatures/Core/Implementation/CommentSelection/CommentUncommentSelectionCommandHandler.cs

```diff
@@ -146,7 +146,7 @@ private ICommentSelectionService GetService(Document document)
                 return service;
             }
 
-            // If we couldn't find one, fallback to the legacy service.
+            // If we couldn't find one, fallback to the  service.
```

Comment:
Accidental search/replace victim?

Reply:
hmm, indeed. haven't found any other like that though

---
In reply to: [158573406](https://github.com/dotnet/roslyn/pull/23769#discussion_r158573406) [](ancestors = 158573406)